pkg/autoops/api/flag_trigger_operation.go

```diff
@@ -60,14 +67,51 @@ func enableFeature(
 	environmentId string,
 	feature *ftdomain.Feature,
 	logger *zap.Logger,
+	publisher publisher.Publisher,
+	editor *eventproto.Editor,
 ) error {
-	if err := feature.Enable(); err != nil {
-		// If the flag is already disabled, we skip the updating
-		return nil
-	}
```

Comment:
I changed the function to use `Update` instead of manually calling it, which increments the flag version automatically.